test/bom.bats

```diff
@@ -1,5 +1,17 @@
 load helpers
 
+function setup_file() {
```

Comment:
setup_file isn't supported in the version of bats that runs in c-i.  afaics c-i just installs 1.2.1 from the ubuntu 22.04 archive.

Even if it was, "zot_setup" is probably something you want to do from "suite_setup" (per test-suite run) rather than setup_file (per file).

I tried to do somethnig similar to this in while avoiding a newer bats dependency at https://github.com/project-stacker/stacker/pull/568/files . i dont' think it actually worked well though.